### ISSUE TITLE
[minor] Cannot close Sorting Install Order modal

### ISSUE THREAD

**Issue by MacSplody** [matortheeternal/mod-picker-issues#39]
![image](https://user-images.githubusercontent.com/1944639/40749052-a51ce21c-645a-11e8-902a-92f42ce9ecef.png)

Only happens when the browser is set to use one half of screen. Just doesn't appear to be responsive at all on desktop. Opera browser.

**Comment by matortheeternal** [comment]
Is there no horizontal scroll bar?  Normally modals can be closed by clicking outside of them, but this modal appears to not have that functionality.

**Comment by MacSplody** [comment]
There are 2 scroll bars, I must have missed it the first time.

![image](https://user-images.githubusercontent.com/1944639/40805627-ccd41d7c-6516-11e8-9b5b-94d38f808912.png)

Top one works fine for modal. Bottom for full page. Modal Scrolls with the bottom one and cannot be closed, must have only seen that at the time.

**Comment by matortheeternal** [comment]
Yeah it's kind of a UX bug that there's two scroll bars, so that should be fixed anyways. :+1:
